openwisp_notifications/websockets/handlers.py

```diff
@@ -1,8 +1,28 @@
 from asgiref.sync import async_to_sync
 from channels import layers
+from django.core.cache import cache
+from django.utils.timezone import now, timedelta
 
 from openwisp_notifications.api.serializers import NotFound, NotificationListSerializer
 
+from ..swapper import load_model
+
+Notification = load_model('Notification')
+
+
+def user_in_notification_storm(user):
+    in_notification_storm = cache.get(f'ow-noti-storm-{user.pk}', False)
```

Comment:
I think we should add a docstring with a summary of how this works, don't you think?